### PR TITLE
(docs) change allowed words in auth.conf to match the actual code

### PR DESCRIPTION
**Pull request by fiddyspence**
The actual puppetserver code does not allow save or destroy as methods

### CODE REVIEW

**Comment by camlow325** [comment]
@fiddyspence Thanks for this PR submission.

The intention of that section of the documentation was to indicate that where one would have used "indirector" names like find, search, save, destroy, etc. as the value for the `method` parameter in rules with the deprecated auth.conf format - like here (https://github.com/puppetlabs/puppet/blob/4.3.1/conf/auth.conf#L118) - one would now need to use the actual HTTP method instead with the new auth.conf format for Puppet Server.  To make this more clear, maybe we should change the header columns on the table to "Indirector name used for deprecated auth.conf" and "Corresponding HTTP method(s) to use for new auth.conf", or something like that.

@gguillotte - can you take a look at this and make a recommendation on how we might reword this to make it more clear?

**Comment by gguillotte** [comment]
@camlow325 The line immediately preceding the table starts with "In the deprecated `auth.conf` format", and the line immediately after the table points users to the deprecated Ruby Puppet `auth.conf` docs. Revising this table's header row and the Ruby docs link could re-emphasize that it's deprecated, but I think we should consider restructuring the `auth.conf` doc to fully segregate the deprecated Ruby content from the supported tk-auth/HOCON content.

**Comment by camlow325** [comment]
> I think we should consider restructuring the auth.conf doc to fully segregate the deprecated Ruby content from the supported tk-auth/HOCON content.

@gguillotte I agree with that suggestion.  We have a separate docs ticket, DOC-2396, which alludes to doing something like this.  For community visibility, would it make sense to you to move (or make a clone of) that ticket in a public project?  I could do either, per your recommendation.

**Comment by gguillotte** [comment]
@camlow325 Good idea. I've moved it to DOCUMENT-466 for now.

I'll close this PR. @fiddyspence, would you like me to give you a heads-up when I have a follow-up PR?
